client/src/components/button/CircleButton.jsx

```
@@ -1,14 +1,17 @@
 
-export default function CircleButton({ clickHandler, content = '좋아요' }) {
-  const icons = { '리뷰 작성': '/edit.svg', '링크 복사': '/link.svg', 좋아요: '/heart.svg' };
+export default function CircleButton({ clickHandler, children = '좋아요' }) {
+  const icons = { '리뷰 작성': edit, '링크 복사': link, 좋아요: heart };
 
   return (
     <div className={styles.wrapper}>
       <button className={styles.circle} onClick={clickHandler}>
-        <img src={icons[content]} alt={content} />
-        <div className={styles.tooltip}>{content}</div>
+        <img src={icons[children]} alt={children} />
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)